src/custom-element.js

```diff
@@ -849,7 +849,8 @@ function createBaseCustomElementClass(win) {
           this.layout_ = applyLayout_(this);
           if (this.layout_ != Layout.NODISPLAY &&
```

Comment:
I'm not sure this should be in here now that we can delay the upgrading of Elements. And, it's just duplication if the upgrade is sync, right? /cc @dvoytenko